src/core/public/chrome/chrome_service.tsx

```diff
@@ -165,12 +167,41 @@ export class ChromeService {
     // Can delete
     const getNavType$ = uiSettings.get$('pageNavigation').pipe(takeUntil(this.stop$));
 
+    const isIE = () => {
+      const ua = window.navigator.userAgent;
+      const msie = ua.indexOf('MSIE '); // IE 10 or older
+      const trident = ua.indexOf('Trident/'); // IE 11
+
+      return msie > 0 || trident > 0;
+    };
+
     if (!this.params.browserSupportsCsp && injectedMetadata.getCspConfig().warnLegacyBrowsers) {
-      notifications.toasts.addWarning(
-        i18n.translate('core.chrome.legacyBrowserWarning', {
-          defaultMessage: 'Your browser does not meet the security requirements for Kibana.',
-        })
-      );
+      notifications.toasts.addWarning({
+        title: mountReactNode(
+          <FormattedMessage
+            id="core.chrome.legacyBrowserWarning"
+            defaultMessage="Your browser does not meet the security requirements for Kibana."
+          />
+        ),
+      });
+
+      if (isIE()) {
+        notifications.toasts.addWarning({
+          title: mountReactNode(
+            <FormattedMessage
+              id="core.chrome.browserDeprecationWarning"
+              defaultMessage="Support for Internet Explorer will be dropped in future versions of this software, please check {link}."
+              values={{
+                link: (
+                  <EuiLink target="_blank" href="https://www.elastic.co/support/matrix" external>
+                    the support matrix on our website
```

Comment:
Should this string be translated as well?

Reply:
Good spot.

Reply:
Fixed.